### PR TITLE
fix: do not reply on env var for built bento/docker

### PR DESCRIPTION
**Pull request by larme**
This will pr write `model_name` `model_id` and `adapter_map` to `_service_vars.py` when building the bento. This means the built bento can be used without providing certain environment variables.

### CODE REVIEW

**Comment by larme** [comment]
I'd like to write all env vars into built bento's `_service_vars.py`, but I think it need more works. So now let's just write most important vars. I will follow with another pr.

File: openllm-python/src/openllm/_llm.py [hunk 1]
```diff
@@ -1122,11 +1117,12 @@ def download():
...
+
   if llm_config is not None:
     attrs.update({
-        'model_id': llm_config['env']['model_id_value'],
+        'model_id': attrs.get("model_id") or llm_config['env']['model_id_value'],
         'quantize': llm_config['env']['quantize_value'],
-        'serialisation': first_not_none(os.environ.get('OPENLLM_SERIALIZATION'), attrs.get('serialisation'), default=llm_config['serialisation']),
+        'serialisation': first_not_none(attrs.get('serialisation'), os.environ.get('OPENLLM_SERIALIZATION'), default=llm_config['serialisation']),
         'system_message': first_not_none(os.environ.get('OPENLLM_SYSTEM_MESSAGE'), attrs.get('system_message'), None),
         'prompt_template': first_not_none(os.environ.get('OPENLLM_PROMPT_TEMPLATE'), attrs.get('prompt_template'), None),
```
Comment (larme) [inline openllm-python/src/openllm/_llm.py]: `model_id` and `serialisation` should be determined at build time

`quantize` maybe should be determined at build time also?

`system_message` and `prompt_template` can be determined at runtime.

**Review by aarnphm — APPROVED** [review]

File: openllm-python/src/openllm/_llm.py (hunk 1 above)
Comment (aarnphm) [reply openllm-python/src/openllm/_llm.py]: I think the env here is for cases when you build a bento with a default system message

Comment (aarnphm) [reply openllm-python/src/openllm/_llm.py]: ofc system message and prompt template can always be modified at runtime

Comment (larme) [reply openllm-python/src/openllm/_llm.py]: ye I think current order is fine

**Pull request merged**